068-persistent-bottom-nav-bar/lib/ui/views/users/users_viewmodel.dart

```diff
@@ -0,0 +1,19 @@
+import 'package:persistent_bottom_nav_bar/app/app.locator.dart';
+import 'package:persistent_bottom_nav_bar/app/router_ids.dart';
+import 'package:persistent_bottom_nav_bar/models/user_model.dart';
+import 'package:persistent_bottom_nav_bar/ui/views/user_details/user_details_view.dart';
+import 'package:stacked/stacked.dart';
+import 'package:stacked_services/stacked_services.dart';
+
+class UsersViewModel extends BaseViewModel {
+  final _navigationService = locator<NavigationService>();
+  List<User> get users => usersData;
+
+  void onUserTap(User user) {
+    _navigationService.navigateWithTransition(
+      UserDetailsView(user: user),
```

Comment:
Same for this. I'd like to see nested routing using named routes and not a mix between Views and widget routes